script/state.py

```diff
@@ -31,4 +33,20 @@
 except Exception as e:
     print(e)
```

Comment:
if there is an exception while writing geojson file, do we need to proceed further. Currently this code allows to proceed

Reply:
We can remove the try except block here, we can add a check on line 19 to check if states data is not present. If none is found we can simply exit after logging an error message.

Reply:
Done